skbio/io/format/genbank.py

```diff
@@ -562,8 +562,7 @@ def _parse_locus(lines):
                ' +(bp|aa|rc)'
                ' +(.*DNA|.*RNA)?'
                ' +(linear|circular)?'
-               ' +(PRI|ROD|MAM|VRT|INV|PLN|BCT|VRL|PHG|'
-               'SYN|UNA|EST|PAT|STS|GSS|HTG|HTC|ENV|CON)'
+               ' +(?!.*DNA|.*RNA)([A-Z]{3})'
```

Comment:
I have to exclude *DNA and * RNA from genbank division, since if molecule type and shape can be empty, and I can't fail the test for missing division (which is expected to fail)